src/dfcx_scrapi/core/intents.py

```diff
@@ -583,22 +584,40 @@ def bulk_intent_to_df(
         agent_id: str = None,
         mode: str = "basic",
         intent_subset: List[str] = None,
+        transpose: bool = False,
         language_code: str = None) -> pd.DataFrame:
         """Extracts all Intents and Training Phrases into a Pandas DataFrame.
 
         Args:
-          agent_id, agent to pull list of intents
-          mode: (Optional) basic returns display name and training phrase as
-            plain text.
-          Advanced returns training phrase and parameters df broken out by
-            parts.
-          language_code: Language code of the intents being uploaded. Ref:
+          agent_id (str):
```

Comment:
Same here. Omit the typing inside the docstring itself in favor of the type hints in the function definition to keep it consistent/clean